panda_gym/pybullet.py

```diff
@@ -108,11 +105,10 @@ def render(
         yaw: float = 45,
         pitch: float = -30,
         roll: float = 0,
-        mode: Optional[str] = None,
     ) -> Optional[np.ndarray]:
         """Render.
 
-        If render mode is "rgb_array", return an RGB array of the scene. Else, do nothing.
+        Return an RGB array of the scene.
```

Comment:
This needs to be changed back. An array is only returned for render more `rgb_array`.